test/unit/lib/plugins/aws/package/lib/addExportNameForOutputs.test.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const chai = require('chai');
+const runServerless = require('../../../../../../utils/run-serverless');
+
+const expect = chai.expect;
+
+describe('test/unit/lib/plugins/aws/package/lib/addExportNameForOutputs.test.js', () => {
+  let outputs;
+
+  before(async () => {
+    const result = await runServerless({
+      fixture: 'aws',
+      command: 'package',
+      configExt: {
+        service: 'myService',
```

Comment:
as the `service` name is generated programmatically for fixtures, it would be better to avoid overriding it unless it's really necessary (I don't think it is in this particular case)

Reply:
Let's instead extract it from `serverless` that is returned by `runServerless` call